CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [1.14.0] - 2023-07-19
```

Comment:
`[1.14.0]` should be rendered as a link to the diff of 1.14.0 and 1.13.2, but in reality it is not.
Please update the bottom part of this file.

```
[Unreleased]: https://github.com/cybozu-go/setup-hw/compare/v1.14.0...HEAD
[1.14.0]: https://github.com/cybozu-go/setup-hw/compare/v1.13.2...v1.14.0
```

This is for so-called reference-style links.

Reply:
I am very appliciate for your suggetion.
I have updated this file.